README.md

```diff
@@ -235,7 +235,12 @@ Use Ctrl-C to stop
 
 ### Releasing new versions
 
-Please refer to the dedicated [page](https://github.com/solidusio/solidus/wiki/How-to-release-extensions) on Solidus wiki.
+1. Ensure that the CI is green
+2. Update the version in `lib/solidus_stripe/version.rb`
+3. Commit the changes with the message "Release vX.Y.Z"
+4. Push the commit
+5. Run `bin/rake release`
+6. Open https://github.com/solidusio/solidus_stripe/releases/vX.Y.Z and generate the changelog
```

Comment:
Yes, I lately use that wiki page only when releasing extensions. BTW, I'm more for starting to move shared things back to `dev_support`, so that all the extensions can benefit from the progress made here and on the other payment gateway extensions. 

In other words, I'm scared to go back to a situation where we need to manually update all the extensions when something happens (change to the sandbox script, or for a dependency constraint).

Reply:
I'm fine with moving stuff back to dev-support, but I thought it wasn't the time yet, and I think we need to think through how we want to handle legacy extensions with regard to that. 

That's why I kept everything local for stripe, so it can be the place from which we can extract the whole recipe, maybe in a new major of dev-support that kinda does things from scratch.

This would probably include sharing some of these scripts and templates with the starter frontend, which poses even more questions on how to deal with things, not being a gem itself.

Reply:
Yes maybe it's not time yet, but can't what we described here for releasing be applied to all extensions by just changing the wiki page?

Reply:
I opened https://github.com/solidusio/solidus_dev_support/issues/202 to track backporting things to dev-support.

This extension is the ultimate blueprint for developing solidus extensions for the starter frontend, and whatever we do here is likely to become the standard and be codified in dev-support, eventually. I also think it's better not to block solidus_stripe on changes to dev-support to avoid both further delays and rushing decisions.

Reply:
@elia, I think we're not being able to communicate our intentions properly :sweat_smile:

> it's better not to block solidus_stripe on changes to dev-support to avoid both further delays and rushing decisions

Absolutely, we're not advocating for that! Our proposal is as follows:

- Copying the following:
```
1. Ensure that the CI is green
2. Update the version in `lib/solidus_stripe/version.rb`
3. Commit the changes with the message "Release vX.Y.Z"
4. Push the commit
5. Run `bin/rake release`
6. Open https://github.com/solidusio/solidus_stripe/releases/vX.Y.Z and generate the changelog
```
- Pasting it into https://github.com/solidusio/solidus/wiki/How-to-release-extensions (replacing `solidus_stripe` by something like `your_extension`).
- Removing the old content in https://github.com/solidusio/solidus/wiki/How-to-release-extensions, maybe adding a line about installing `gem-release` if not present.
- Remove the instructions from this PR and just have a link to the Solidus wiki.

Your new proposal is better. As you said, with GH releases and the automatic release notes generator, everything is a piece of cake. There's no need to use `solidus_dev_support` for any extension. And we want to avoid having instructions that can differ from one extension to another.

I hope this is clear now :slightly_smiling_face: What do you think about it?

Reply:
@waiting-for-dev I was under the impression there were multiple positions on this, thanks for clarifying 🙌 

The wiki entry is up to date, I also added instructions on how to migrate older extensions and kept the older instructions under a `<summary>` in case there's no time to migrate an extension.